GitHubContents.ps1

```diff
@@ -358,7 +385,14 @@ filter Set-GitHubContent
 
     $uriFragment = "/repos/$OwnerName/$RepositoryName/contents/$Path"
 
-    $encodedContent = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($Content))
+    if (-not [String]::IsNullOrWhitespace($ContentPath))
+    {
+        $encodedContent = [Convert]::ToBase64String([System.IO.File]::ReadAllBytes($ContentPath))
```

Comment:
Now that you aren't accessing the file directly through a PowerShell function, you need to proactively resolve this filepath, certain filepaths won't be processed correctly when passed-in to a .NET function like this unless we ensure it's full resolved.

I'd suggest you do this first:

```suggestion
        $ContentPath = Resolve-UnverifiedPath -Path $ContentPath
        $encodedContent = [Convert]::ToBase64String([System.IO.File]::ReadAllBytes($ContentPath))
```

Reply:
That makes sense